### PR TITLE
Add support for CSS font-weight + sample app for testing fonts

### LINKED ISSUE #1862: Support for all font-weight CSS values with the best mapping possible.
Support those values:
 - normal (same as 400)
 - bold (same as 700)
 - 100 (Thin)
 - 200 (Extra Light / Ultra Light)
 - 300 (Light)
 - 400 (Normal)
 - 500 (Medium)
 - 600 (Semi Bold / Demi Bold)
 - 700 (Bold)
 - 800 (Extra Bold / Ultra Bold)
 - 900 (Black / Heavy)

[iOS Font Weights](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIFontDescriptor_Class/index.html#//apple_ref/doc/uid/TP40013284)

[Android Font Weights](http://stackoverflow.com/questions/19691530/valid-values-for-androidfontfamily-and-what-they-map-to)

### PR DESCRIPTION
**Pull request by hamorphis**
Resolves #1862

### CODE REVIEW

**Comment by hamorphis** [comment]
run ci

**Comment by hamorphis** [comment]
run ci

**Comment by hamorphis** [comment]
run ci

**Comment by hamorphis** [comment]
run ci

File: ui/enums/enums.d.ts [hunk 1]
```diff
@@ -485,14 +485,49 @@
      */
     export module FontWeight {
         /**
-         * Normal font weight.
+         * Thin font weight. CSS font-weight 100.
```
Comment (hshristov) [inline ui/enums/enums.d.ts]: Do we need to export all of these if they are not cross-platform? Why not only few - light & exxtraBold?

**Comment by hamorphis** [comment]
run ci

**Comment by vekexasia** [comment]
hello @hamorphis should this be used like so? 

```
Label {
 font-weight: 100;
}
```

cause i cant see any effect on my app.

**Pull request merged**